Initial XR support (#3245)
f3d-app/f3d: doc/user/03-OPTIONS.md

Filename to save [screenshots](04-INTERACTIONS.md#taking-screenshots) to. Can use [template variables](#filename-templating). Supports relative paths [as described](04-INTERACTIONS.md#taking-screenshots).

> ### `--rendering-backend=<auto|egl|osmesa|glx|openxr|wgl>` (_string_, default: `auto`)

Reviewer: openxr or xr ?

> Rendering backend to load, `auto` means to let F3D pick the correct one for you depending on your system capabilities. Use `egl` or `osmesa` on linux to force headless rendering. Use `xr` to launch F3D with `xr` rendering backend.

Reviewer 2: EGL/OSMesa isn't Linux only anymore. Not related to your change but please change this. Also let's try to be more explicit about `xr`.
```suggestion
Rendering backend to load, `auto` means to let F3D pick the correct one for you depending on your system capabilities. Use `egl` or `osmesa` to force headless rendering. Use `xr` to launch F3D in XR mode.
```

Author: Will update the EGL/OSMesa.

https://github.com/f3d-app/f3d/pull/3245#discussion_r3796747246 @Meakk @mwestphal 
should I use XR mode or xr rendering backend for doc?

Reviewer 2: It feels wrong to me to say "Use rendering-backend=xr to specify F3D should use the xr rendering backend". It must be obvious to the user what it is. What about "Use `xr` to launch F3D in Virtual or Augmented Reality context"?

> Rendering backend to load, `auto` means to let F3D pick the correct one for you depending on your system capabilities. Use `egl` or `osmesa` to force headless rendering. Use `rendering-backend=xr` to specify F3D should use the xr rendering backend

Reviewer: ```suggestion
Rendering backend to load, `auto` means to let F3D pick the correct one for you depending on your system capabilities. Use `egl` or `osmesa` to force headless rendering. Use `rendering-backend=xr` to specify F3D should use the xr rendering backend, refered in this doc as "XR mode".
```
